src/main/kotlin/lotto/application/LottoApplication.kt

```diff
@@ -0,0 +1,7 @@
+package lotto.application
+
+import lotto.view.LottoController
+
+fun main() {
+    LottoController.payoutResult(LottoController.lottoPurchased(), LottoController.lottoWinnerNumber())
```

Comment:
한라인에 많은 기능이 있다면 오히려 가독성이 떨어질 수 있다고 생각하는데요!

아래와 같이 라인을 좀 나눠보면 어떨까요?

```suggestion
    val purchasedLottoResults = LottoController.lottoPurchased()
    val lottoWinnerNumbers = LottoController.lottoWinnerNumber()
    LottoController.payoutResult(purchasedLottoResults, lottoWinnerNumbers)
```

Reply:
아고, 이 부분도 이전 미션과 동일하게 코드 라인 수를 의식하다보니 오히려 가독성을 해치는 코드가 되었네요ㅠ

말씀해주신 것에 전적으로 동의해 제안 주신 코드로 수정하였습니다!